UI/PauseScreen.cpp

```diff
@@ -176,17 +182,58 @@ class ScreenshotViewScreen : public PopupScreen {
 	bool ShowButtons() const override { return true; }
 
 	void CreatePopupContents(UI::ViewGroup *parent) override {
-		UI::LinearLayout *content = new UI::LinearLayout(UI::ORIENT_VERTICAL);
-		parent->Add(content);
-		UI::Margins contentMargins(10, 0);
-		content->Add(new AsyncImageFileView(filename_, UI::IS_KEEP_ASPECT, new UI::LinearLayoutParams(480, 272, contentMargins)))->SetCanBeFocused(false);
+		using namespace UI;
+		auto pa = GetI18NCategory("Pause");
+		auto di = GetI18NCategory("Dialog");
+
+		ScrollView *scroll = new ScrollView(ORIENT_VERTICAL, new LinearLayoutParams(FILL_PARENT, 50, 1.0f));
+		LinearLayout *content = new LinearLayout(ORIENT_VERTICAL);
+		Margins contentMargins(10, 0);
+		content->Add(new AsyncImageFileView(filename_, IS_KEEP_ASPECT, new LinearLayoutParams(480, 272, contentMargins)))->SetCanBeFocused(false);
+		GridLayoutSettings gridsettings(240, 64, 5);
+		gridsettings.fillCells = true;
+		GridLayout *grid = content->Add(new GridLayoutList(gridsettings, new LayoutParams(FILL_PARENT, WRAP_CONTENT)));
+		grid->Add(new Choice(pa->T("Save State")))->OnClick.Handle(this, &ScreenshotViewScreen::OnSaveState);
+		grid->Add(new Choice(pa->T("Load State")))->OnClick.Handle(this, &ScreenshotViewScreen::OnLoadState);
+		Choice *back = new Choice(di->T("Back"));
+		grid->Add(back)->OnClick.Handle<UIScreen>(this, &UIScreen::OnBack);
+		Choice* undoButton = new Choice(pa->T("Undo State"));
```

Comment:
nit: `Choice *`.  I think lowercasing the second work makes sense, but perhaps we should use "Undo save" or "Undo last save" (not sure that will fit.)

Currently there's no undo for loading state, but I could see adding it... not sure it'd make sense to use the same button though?  I feel like I'd want that to be a button on the main pause screen, if the feature existed.

Btw this section looks pretty "dense"... maybe could add a newline before setting up the frid settings and then after setting up the buttons, and put all the `grid->Add` together?  Small thing, but it initially looked like a wall of code.

-[Unknown]